new_project/examples/separate_apps/src/maker.ts

```diff
@@ -24,13 +56,29 @@ import { checkEnvFile, startClient } from "./lib";
     console.log("[Maker] peer id:", peerId);
     console.log("[Maker] address hint:", addressHint);
 
-    // Note that we assume the maker published an offer
-    // somehow somewhere and that someone (a taker)
-    // will take such offer by send a SWAP request to the
-    // maker's COMIT node. Publishing and Finding offers
-    // is not part of this example.
+    // start negotiation protocol handler so that a taker can take the order and receives the latest rate
+
+    const negotiationProtocolHandler = new NegotiationProtocolHandler();
+    negotiationProtocolHandler.start(2318); // CoBloX Founding Date 🚀
+    const order: Order = {
+        ...defaultOrder,
+        execution_params: {
+            connection_info: {
+                peer_id: peerId,
+                address_hint: addressHint,
+            },
+            expiries: {
+                ask_expiry: moment().unix() + 7200,
+                bid_expiry: moment().unix() + 3600,
+            },
+            role: "alice",
```

Comment:
🙃 This is a bit confusing as well. I am not sure the role should be part of the offer. I mixes the execution into the negotiation again. That is what we are trying to avoid I feel.

In my understanding the other party (Taker) does see the price-relevant things in the order, but does not see the other execution related parameters. The role should not be part of the order for me.

Reply:
I'm not sure if I follow your comment. Can you clarify what you mean with : 
> In my understanding the other party (Taker) does see the price-relevant things in the order, but does not see the other execution related parameters.

Where does he `see` things and where not?